### PR TITLE
fix(label): update label properties

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Enhancement

#### What is expected?

I updated Label component in order to follow tokens.

### CODE REVIEW

File: src/components/Text/index.tsx [hunk 1]
```diff
@@ -81,40 +81,42 @@ type TextProps = {
...
+    disabled: boolean
+    italic: boolean
+    underline: boolean
+  }>(generateStyles)
 
 const Text = ({
   variant,
   children,
-  as,
+  as = 'div',
```
Comment (chambo-e) [inline src/components/Text/index.tsx:119]: I assumed we always wanted to specify it ?

File: src/components/Text/index.tsx [hunk 2]
```diff
@@ -136,9 +139,11 @@ const Text = ({
     }
   }, [oneLine])
 
+  const StyledTextComponent = StyledText(as)
```
Comment (chambo-e) [inline src/components/Text/index.tsx:142]: Wuut ? That is really ugly, and kinda dangerous this

No way to have it work with the native styled as ?

Comment (matthprost) [reply src/components/Text/index.tsx:142]: @chambo-e The problem with native one is that component doesn't accept any others props that the ones of the default tag (here it as a `div`) so `htmlFor` won't be accepted as it doesn't exist in div properties. That's the only workaround I've found for this :/

File: src/components/Text/index.tsx (hunk 1 above)
Comment (matthprost) [reply src/components/Text/index.tsx:119]: @chambo-e You're right I was confused by control value in storybook

**Review by philibea — APPROVED** [review]

**Review by vincentaudebert — APPROVED** [review]

**Pull request merged**